packages/gafl-webapp-service/src/pages/summary/contact-summary/__tests__/route.spec.js

```diff
@@ -33,349 +36,278 @@ const route = require('../route.js').default
+    const permission = getMockPermission(
+      {},
+      {
+        licenceLength: '1D',
+        isLicenceForYou
+      }
+    )
+    const mssgCatalog = getMockCatalog({
+      [mssgKey]: mssg
     })
-
-    describe('when purchasing a 12 month (physical licence) without postal fulfilment', () => {
-      it('should display the Licence and Contact as the email and Newsletter as no', async () => {
-        const permission = {
-          licenceLength: '12M',
-          licensee: {
-            ...addressAndContact,
-            postalFulfilment: false,
-            preferredMethodOfConfirmation: 'Email',
-            preferredMethodOfReminder: 'Email',
-            email: 'new3@example.com',
-            preferredMethodOfNewsletter: 'Prefer not to be contacted'
-          }
-        }
-        const sampleRequest = generateRequestMock(permission)
-
-        const { summaryTable } = await getData(sampleRequest)
-
-        expect(summaryTable).toMatchSnapshot()
-      })
-
-      it('should display the Licence and Contact as the phone number and Newsletter as yes', async () => {
-        const permission = {
-          licenceLength: '12M',
-          licensee: {
-            ...addressAndContact,
-            postalFulfilment: false,
-            preferredMethodOfConfirmation: 'Text',
-            preferredMethodOfReminder: 'Text',
-            mobilePhone: '07700900900',
-            preferredMethodOfNewsletter: 'Text'
-          }
-        }
-        const sampleRequest = generateRequestMock(permission)
-
-        const { summaryTable } = await getData(sampleRequest)
-
-        expect(summaryTable).toMatchSnapshot()
-      })
+    const sampleRequest = getRequestMock({
+      permission,
+      catalog: mssgCatalog
     })
+    const { changeLicenceDetails } = await getData(sampleRequest)
+    expect(changeLicenceDetails).toBe(mssg)
+  })
 
-    describe('when purchasing a 1 day (non physical licence)', () => {
-      it('should display the Contact as the email and Newsletter as no', async () => {
-        const permission = {
-          licenceLength: '1D',
-          licensee: {
-            ...addressAndContact,
-            preferredMethodOfConfirmation: 'Email',
-            preferredMethodOfReminder: 'Email',
-            email: 'new3@example.com',
-            preferredMethodOfNewsletter: 'Prefer not to be contacted'
-          }
-        }
-        const sampleRequest = generateRequestMock(permission)
-
-        const { summaryTable } = await getData(sampleRequest)
-
-        expect(summaryTable).toMatchSnapshot()
+  it.each([
+    { preferredMethodOfNewsletter: HOW_CONTACTED.email, expectedReturn: mssgs.yes, action: 'sent' },
+    { preferredMethodOfNewsletter: HOW_CONTACTED.text, expectedReturn: mssgs.yes, action: 'sent' },
+    { preferredMethodOfNewsletter: HOW_CONTACTED.post, expectedReturn: mssgs.yes, action: 'sent' }
```

Comment:
It looks like you've removed the only case that tested the reverse (`[HOW_CONTACTED.none, mssgs.no]`), so now `expectedReturn` and `action` don't vary at all. You needed to keep that original case, so put it back in. Then check that you're not passing in something that never varies - if you are that can be hard-coded into the test.

Reply:
Unsure why these have no updated sorry, as these are out of date.